frontend/src/theme/components/Tabs.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  ChakraTheme,
+  ComponentMultiStyleConfig,
+  SystemStyleObjectRecord,
+  ThemingPropsThunk,
+} from '@chakra-ui/react'
+import { getColor } from '@chakra-ui/theme-tools'
+import merge from 'lodash/merge'
+
+export const DRAG_SCROLL_SPEED = 0.8
+
+const variantLine: ThemingPropsThunk<SystemStyleObjectRecord, ChakraTheme> =
+  () => {
+    return {
+      root: {
+        w: '100%',
+      },
+      tablist: {
+        w: '100%',
+        overflowX: 'scroll',
+        borderColor: 'transparent',
+        /* Hide scrollbar for IE and Edge */
+        '&::MsOverflowStyle': 'none',
+        /* Hide scrollbar for Firefox */
+        '&::scrollbarWidth': 'none',
+        /* Hide scrollbar for Chrome, Safari and Opera */
+        '&::-webkit-scrollbar': {
+          display: 'none',
+        },
+        p: '1.5625rem',
```

Comment:
This number seems a little off. We use a 4 point grid (even numbers), and looking at the figma mock the padding seems to be 28px top 22px bottom. Can you double check all the margin figures again?

The total height of this tab bar is also 80px, but 84px in your story.